packages/x-components/src/plugins/x-emitters.ts

```diff
@@ -6,6 +6,7 @@ import { AnySimpleStateSelector, AnyStateSelector } from '../store/utils/store-e
 import { XEventPayload, XEventsTypes } from '../wiring/events.types';
 import { WireMetadata } from '../wiring/wiring.types';
 import { AnyXModule } from '../x-modules/x-modules.types';
+import { WatchOptionsBase } from 'vue';
```

Comment:
estlint complains about the import order, we should move it to above the relative ones